frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1212,13 +1212,20 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
     }
 
     private void prepareStandardView(RenderRequest request) throws IOException {
-        User user = UserCacheHolder.getUserFromRequest(request);
-        List<Organization> organizations = UserUtils.getOrganizations(request);
-        request.setAttribute(PortalConstants.ORGANIZATIONS, organizations);
-        request.setAttribute(IS_USER_ADMIN, PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user) ? YES : NO);
-        for (Project._Fields filteredField : projectFilteredFields) {
-            String parameter = request.getParameter(filteredField.toString());
-            request.setAttribute(filteredField.getFieldName(), nullToEmpty(parameter));
+        List<Project> projectList;
+        try {
+            User user = UserCacheHolder.getUserFromRequest(request);
+            ProjectService.Iface projectClient = thriftClients.makeProjectClient();
+            projectList = projectClient.getAccessibleProjectsSummary(user);
+            Set<String> organizations = getProjectGroups(projectList);
+            request.setAttribute(PortalConstants.ORGANIZATIONS, organizations);
+            request.setAttribute(IS_USER_ADMIN, PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user) ? YES : NO);
+            for (Project._Fields filteredField : projectFilteredFields) {
+                String parameter = request.getParameter(filteredField.toString());
+                request.setAttribute(filteredField.getFieldName(), nullToEmpty(parameter));
+            }
+        }catch(TException e) {
```

Comment:
Indentation. `}catch`
Also kindly don't put entire method in try-catch.
I guess only this part should be placed in try-catch - `projectList = projectClient.getAccessibleProjectsSummary(user);`